lib/map/security_token/configuration.rb

```diff
@@ -78,6 +90,21 @@ def client_assertion_certificate
         OpenSSL::X509::Certificate.new(File.read(client_cert_path))
       end
 
+      def public_jwks
+        @public_jwks ||= Rails.cache.fetch(jwks_cache_key, expires_in: jwks_cache_expiration) do
+          response = connection.get(provider_jwks_path)
+          Rails.logger.info("#{logging_prefix} Get Public JWKs Success")
+
+          parse_public_jwks(response:)
+        end
+      end
+
+      def parse_public_jwks(response:)
+        keys_array = response.body['keys']
```

Comment:
Is there a reason this is different thank the 'public_jwks.rb' implementation?

>      jwks = JWT::JWK::Set.new(response.body)
>      jwks.select! { |key| key[:use] == 'sig' }
>      jwks

It seems like this implementation is the same but just slightly simpler